contrib/gitian-descriptors/gitian-win.yml

```diff
@@ -2,7 +2,7 @@
 name: "bitcoin-win-0.17"
 enable_cache: true
 suites:
-- "trusty"
+- "bionic"
 architectures:
 - "amd64"
 packages:
```

Comment:
Should rename "git-core" to "git"?

Reply:
Yes, I already do.

Reply:
In line 11 of this file in the current version it still says "git-core", no?

Reply:
Oh, I missed that one.